submissions/NadiaVorontsova/hooli-style-popup/index.html

```diff
@@ -0,0 +1,238 @@
+                                                class="header__popup__menu__img">
+                                            <p>Videos</p>
+                                        </div>
+                                    </a>
+                                </li>
+                                <li class="header__popup__menu__item">
+                                    <a href="#" class="header__popup__menu__link link">
+                                        <div class="header__popup__menu__container">
+                                            <img src="assets/search.png" width="48" height="35" alt=""
+                                                class="header__popup__menu__img">
+                                            <p>Search</p>
+                                        </div>
+                                    </a>
+                                </li>
+                                <li class="header__popup__menu__item">
+                                    <a href="#" class="header__popup__menu__link link">
+                                        <div class="header__popup__menu__container">
+                                            <img src="assets/photos.png" width="48" height="35" alt=""
+                                                class="header__popup__menu__img">
+                                            <p>Photos</p>
+                                        </div>
+                                    </a>
+                                </li>
+                                <li class="header__popup__menu__item">
+                                    <a href="#" class="header__popup__menu__link link">
+                                        <div class="header__popup__menu__container">
+                                            <img src="assets/apps.png" width="48" height="35" alt=""
+                                                class="header__popup__menu__img">
+                                            <p>Apps</p>
+                                        </div>
+                                    </a>
+                                </li>
+                                <li class="header__popup__menu__item">
+                                    <a href="#" class="header__popup__menu__link link">
+                                        <div class="header__popup__menu__container">
+                                            <img src="assets/documents.png" width="48" height="35" alt=""
+                                                class="header__popup__menu__img">
+                                            <p>Documents</p>
+                                        </div>
+                                    </a>
+                                </li>
+                            </ul>
+                            <input type="checkbox" id="popup_checkbox_more" class="header__popup__checkbox__more">
```

Comment:
Should be possibility to show hidden menu by space click on keyboard

Reply:
+